src/main/java/org/openhab/tools/analysis/checkstyle/DeclarativeServicesDependencyInjectionCheck.java

```diff
@@ -44,6 +44,11 @@
     }
 
     @Override
+    public int[] getAcceptableTokens() {
+        return getDefaultTokens();
```

Comment:
Could it be that the default tokens are too much or too less for this check? Why not specifying only the needed ones?